ydb/docs/en/core/reference/ydbops/rolling-restart-scenario.md

```diff
@@ -0,0 +1,92 @@
+# Performing {{ ydb-short-name }} cluster restart using ydbops
```

Comment:
Generally I would expect 2 section in ToC for ydbops:
1. Scenarios
2. Reference

I suppose many details should go to reference